docfx_yaml/extension.py

```diff
@@ -27,6 +27,11 @@
 from .monkeypatch import patch_docfields
 from .directives import RemarksDirective, TodoDirective
 from .nodes import remarks
+from sphinx.addnodes import toctree
+from sphinx.util.logging import getLogger 
```

Comment:
Since we have to pin the Sphinx version to 1.5.5, importing logger like this will fail.